common/components/utils/NavigationLinks.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+
+import metrics from "./metrics.js";
+import CurrentUser from "./CurrentUser.js";
+
+type DLAB_NAV_LINK = {|
+  u: string,
+  n: string,
+  isButton: ?boolean
+|};
+
+export type NavigationLink = {|
+  url: string,
+  name: string,
+  isButton: boolean
+|};
+
+const siteNavLinks =
```

Comment:
No need to use this intermediate object anymore, just use NavigationLink objects.